Register product features for JDR reports

israel-hdez opened the pull request:
JDR report fetures are registered as an admin feature under middleware
servers.

Jira: https://issues.jboss.org/browse/HAWKULAR-1231

abonas commented:
@miq-bot add_label enhancement, providers/hawkular
LGTM
@miq-bot assign @chessbyte 
@chessbyte who would be the best person to review/merge this?

abonas commented:
@chessbyte who would be the best person to review/merge this?

h-kataria commented:
@israel-hdez is there a related UI PR for these new features?

israel-hdez commented:
@h-kataria Yes, this one: https://github.com/ManageIQ/manageiq-ui-classic/pull/1764

h-kataria commented:
@israel-hdez i only see Generate report button on that PR https://github.com/ManageIQ/manageiq-ui-classic/pull/1764/files#diff-e6f07e5cdde7794bbc9e7cac60727879R187 , is Delete button in a different PR?

israel-hdez commented:
@h-kataria  Oh! I moved the delete button from main toolbar to a secondary toolbar. It's, indeed, a good catch in the UI. I just updated the PR and now you can find it here: https://github.com/ManageIQ/manageiq-ui-classic/pull/1764/files#diff-002598c1d95530eae2e85c719166439fR27.

bronaghs commented:
@h-kataria - look good to go?

chessbyte commented:
@h-kataria I can merge with your approval.

h-kataria commented:
@chessbyte looks good, please merge.
